src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const grouped: GroupsMap<T> = {};
+
+  items.forEach((item) => {
+    const sorted = String(item[key]);
```

Comment:
I don't know if it is necessary, maybe you need to add a type for the argument item and for sorted

Reply:
VSCode doesn't know about that problem)